niworkflows/__init__.py

```diff
@@ -19,3 +20,5 @@
 
 NIWORKFLOWS_LOG = logging.getLogger(__packagename__)
 NIWORKFLOWS_LOG.setLevel(logging.INFO)
+
+matplotlib.use('Agg')
```

Comment:
This sets the matplotlib backend. Without it I was not able to use matplotlib on circleci or on my mac. The legacy image generation code from Craig actually had this sprinkled in each of the plotting functions. On circle the error message generated had to with the fact that the version of python being used did not have the _tkinter module, but I don't have the exact text of it at hand unfortunately.